src/components/Tooltip/Tooltip.mdx

```diff
@@ -77,13 +98,12 @@ aspect ratio to not render properly.
 
 <Canvas of={TooltipStories.OnLinkedIcons} />
 
-### Tooltip with Icon Content
-
-<Canvas of={TooltipStories.UsingIconContent} />
+### Icon and Image Content
 
-### Tooltip with Image Content
+While the Reservoir Tooltip component does allow icons and images to be included
```

Comment:
Nit for brevity: While the Reservoir `Tooltip` does allow icons and images to be included
in its content, this practice should be avoided.